aboutpage.html

```diff
@@ -0,0 +1,140 @@
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </div>
+      </div>
+
+      <section class="invitation">
+        <div class="topic">
+          <h3 class="inv-header2">"Hello ladies and gentlemen!"</h3>
+          <h1 class="slogan">New Year's sales!</h1>
+          <p id="program" class="home-para">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nisi quia, cum esse nemo minima excepturi in quas quibusdam ut aperiam dolor! Velit tempore, asperiores eius ipsum illum blanditiis dolorem itaque?
+          </p>
+        </div>
+        <div class="contact">
+          <h2 class="contact-us">Please contact us per Email for further questions about 2023 exhibition!</h2>
+          <p id="join" class="cont-para">
+            Email: <a href="#">rolandntwali@gmail.com</a>
+          </p>
+        </div>
+      </section>
+    </header>
+<section class="main">
+      <div class="ex-logo">
+        <h3 class="ex-logo-h">
+          Wood and Words logo
+        </h3>
+        <hr class="hr-1" />
+        <p id="news" class="ex-logo-para">
+         Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sequi voluptas, sapiente facere nihil ratione laudantium porro laborum molestias eligendi, voluptate soluta beatae debitis suscipit vitae. Similique cupiditate nemo eveniet illo?
+        </p>
+
+        <div class="ww-logo">
+          <img
+            class="ww-logo-design"
+            src="./images/Logo 10.png"
+            alt="Wood and Words"
+          />
+        </div>
+      </div>
+     <hr class="about-line">
+      <div class="past-market">
+        <h3 class="past-market-h">
+          See the past New year's sales
+          
+        </h3>
+        <hr class="hr-1" />
+        <p class="past-market-p">
+          Take a look at the last two markets which took place city center.
+        </p>
+
+        <div id="exhibition" class="both-images">
+          <div class="image-1">
+            <p class="img-parag">
+              <span>2022 </span><br />
+              Made in Rwanda exhibiton
+            </p>
+          </div>
+
+          <div class="image-2">
+            <p class="img-parag">
+              <span>2021 </span> <br />
+              Made in Rwanda exhibiton
+            </p>
+          </div>
+        </div>
+      </div>
+
+    <div id="sponsor" class="partners">
+      <h3 id="p-title">Partners</h3>
+      <hr class="hr-1" />
+      <div class="partners-c">
+        <img id="wood" src="./images/Logo 10.png" alt="Wood and words" />
+        <img id="toddle-care" src="./images/tc.jpg" alt="Toddle care" />
+      </div>
+    </div>
+  </section>
+  <footer class="footer">
+    <img src="./images/RO.jpeg" alt="" class="footer-img" />
+    <p class="footer-para">
+      2023 Made in Rwanda Exhibition. <br />Some Rights Reserved.
+    </p>
+  </footer>
+
+    <script src="./aboutpage.js"></script>
```

Comment:
- [ ] Nice work you did here. however, the background on the desktop should be dark based on the figma design provided. I've attached a photo comparison to show you the difference. sticking to the design  is important as that is how potential clients or your future managers will ask for 👏 

# Your view:
![image](https://user-images.githubusercontent.com/50990925/217795404-ce5e1af8-209a-46ea-acc8-4640aa641a44.png)


# Figma:
![image](https://user-images.githubusercontent.com/50990925/217795163-b24e2140-9cc8-4b63-a202-8527227ce005.png)